src/govuk/components/footer/template.njk

```diff
@@ -1,6 +1,6 @@
 {% from "../../macros/logo.njk" import govukLogo -%}
 
-{%- set _rebrand = params.rebrand | default(govukRebrand() if govukRebrand is callable else govukRebrand) -%}
+{%- set _rebrand = params.rebrand | default(govukRebrand) -%}
```

Comment:
I guess this is change because we've made the call not to port the Prototype Kit plugin to v4?

Would be good to capture this in the commit message as at the minute it's not clear how this change relates.